app/controllers/sign_up/completions_controller.rb

```diff
@@ -0,0 +1,24 @@
+module SignUp
+  class CompletionsController < ApplicationController
```

Comment:
That won't be enough. Consider this scenario:

1. I am already signed up but not proofed yet
2. I come in via an SP that makes an LOA3 request
3. I get redirected to the proofing flow
4. Instead of starting the proofing flow, I visit the `/sign_up/completed` endpoint
Result: I see "You have verified your identity with login.gov", which is not true. The text that is displayed only looks at whether or not the SAML authentication context was LOA 3 or not. It does not take into account the user's status.